frontend/catalyst/utils/jax_cpu_lapack_kernels/lapack_kernels.h

```diff
@@ -0,0 +1,215 @@
+/* Copyright 2021 The JAX Authors.
```

Comment:
I think the header should still be Xanadu but then cite the Jax copyright? @josh146

Reply:
Yep :+1: Because we use compatible licenses (we both use Apache 2.0), what you suggest @dime10 (under the Xanadu copyright header) looks great.